packages/zoe/test/swingsetTests/zoe/bootstrap.js

```diff
@@ -115,6 +117,14 @@ function build(E, log) {
           autoswapBundle.source,
           autoswapBundle.moduleFormat,
         ),
+        sellItems: await E(zoe).install(
```

Comment:
In the new-SES branch, this pattern changes to just `.install(sellItemsBundle)`, same with `mintAndSellNFTBundle`